crates/router/src/connector/bluesnap.rs

```diff
@@ -504,84 +504,6 @@ impl api::PaymentSession for Bluesnap {}
 impl ConnectorIntegration<api::Session, types::PaymentsSessionData, types::PaymentsResponseData>
     for Bluesnap
 {
-    fn get_headers(
```

Comment:
@Sangamesh26 Do not remove this code, let it be and add comments. Revert only the core changes